### PR TITLE
[Ready] edited try for free button text

### PR DESCRIPTION
**Pull request by GerwinV**
Merge if you like it, kill it if you dont!
@dkhgh

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ en:
     head_title: "Trial Signup - LearningSpaces"
     head_description: "Get a free trial for LearningSpaces, the fun and simple to use learning platform to enable informal learning and knowledge sharing among peers."
     page: https://public.learningspaces.io/accounts/new
-    request: Try LearningSpaces for Free
+    request: try learningspaces free for 60 days
```
Comment (dkhgh) [inline locales/en.yml]: LearningSpaces :cactus:

Comment (Marthyn) [inline locales/en.yml]: lowercase :fearful:

Comment (GerwinV) [inline locales/en.yml]: its al upcased, so figured it didnt matter ;-)

**Comment by tarzan** [comment]
what about this one?

**Comment by dkhgh** [comment]
Forgot about it, IMO more something for A/B testing with [VWO](http://vwo.com).

**Comment by fatboypunk** [comment]
VWO seems pretty simple to do, so lets do that.

**Pull request closed without merging**